trulens_eval/trulens_eval/react_components/record_viewer/dist/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Record viewer</title>
-    <script type="module" crossorigin src="./assets/index-be726982.js"></script>
+    <script type="module" crossorigin src="./assets/index-b9ca4089.js"></script>
```

Comment:
@piotrm0 I think the new generated js file might not be added to this PR

Reply:
Added.